src/App.tsx

```diff
@@ -43,6 +84,8 @@ function App() {
     sourceID,
   });
 
+  if (!isOnClientSide()) return null;
```

Comment:
So just to get this right - when SSRing we actually don't render anything (btw. this results in React warning that gets logged in the console when rehydrating). Which parts of this component are relevant for SSR then?

Reply:
Correct. But not sure I understand the question?

Reply:
Is it important for SSR to execute anything in this component? I mean - could the `isOnClientSide` check be moved to the top of this component? 

And also - what we can do about React warning?

Reply:
We could move it to above this component in the tree? I.e. `<NoSSR><App /></NoSSR>`. I don't like this because I like the idea you could use `<App />` without caring if it needs to be SSR'd or not.

The reason it's at the bottom is so we don't conditionally render hooks.

We could make a HOC?

`withNoSSR(App)`

But I prefer the above implementation.

Reply:
> The reason it's at the bottom is so we don't conditionally render hooks.

Forgot about this pesky thing. It's fine how it is then.